lib/dependency_bumper/updater.rb

```diff
@@ -46,6 +46,8 @@ def create_git_branch(report)
 
       git_repo.add(all: true)
       git_repo.commit(output)
+    rescue Git::GitExecuteError => error
+      Console.logger.error(error)
```

Comment:
Originally, the commit was failing for me because of the GPG signing key prompt but the error message wasn't very helpful. This dumps the full error which isn't great but at least it makes it a little easier to spot where the problem is.